src/game/component/TransformComponent.h

```diff
@@ -25,9 +25,9 @@ class TransformComponent : public IComponent
 			  const glm::vec3& scale = glm::vec3(1,1,1));
 	void Refresh(){}
 	const glm::mat4& GetWorldMatrix() const;
-	const glm::mat4& GetCameraMatrix() const;
 
-	inline void MarkDirty(){ m_PosMatrix[3][0] = 1.f; }
+	inline void MarkDirty(){ m_PosMatrix[0][3] = 1.f; }
```

Comment:
`inline void MarkDirty(){ m_PosMatrix[3][0] = 1.f; }`
changed to
`inline void MarkDirty(){ m_PosMatrix[0][3] = 1.f; }`

I don't understand the purpose of this change. Could you please elaborate?